content/en/docs/started/OWNERS

```diff
@@ -1,3 +1,9 @@
 approvers:
   - RFMVasconcelos
-  
+  - 8bitmp3
+
+reviewers:
+  - Bobgy
+  - knkski 
+  - RFMVasconcelos
+  - 8bitmp3
```

Comment:
It's better not to duplicate reviewer and approver to make sure everyone is notified